### ISSUE TITLE
Twitter Bootstrap support

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2334]
Originally reported on Google Code with ID 2069

```
Hi,

Twitter Bootstrap is more and more popular, so could you please add the ability to
use Bootstrap's classes instead of yours ?

With this option it will be easier to implement FullCalendar in an existing (bootstrap)
project. 
Also using default classes will allow people to use all themes already available (from
http://bootswatch.com/ , https://wrapbootstrap.com/?ref=bsw or any others)...

I'm not a CSS/jQuery/Bootstrap ninja but you probably can do it by defining all your
classes in a specific files that must be included before the main script. One files
will use Bootstrap classes, an other one will use jQuery UI, etc...

My english is really poor so I hope all of this is readable !

Thanks !
```

Reported by `schmittounet` on 2014-01-13 14:12:10

Imported with **6** stars.

**Comment by arshaw** [comment]
this is queued for the next release. see https://github.com/fullcalendar/fullcalendar/pull/3566. closing

**Comment by arshaw** [comment]
Released in v3.5.0 ([blog post](https://fullcalendar.io/blog/2017/08/bootstrap-support-and-future-developments/#more-257))
